### ISSUE TITLE
Issues with dark mode on Microsoft edge

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1553]
Here's the page as it looks normally:
![VC4IT-correct](https://github.com/user-attachments/assets/df4cefc4-457d-4358-871b-e80a2f6cc53e)

Here's MS Edge dark mode:
![VC4IT-incorrect](https://github.com/user-attachments/assets/20e59c5d-7f18-48b0-bb5c-f843effb41ca)

**Comment by jcsparks** [comment]
I don't believe that we have any dark-mode specific code in our theme (other than the one advanced design option for specific articles).  We can't control browser-specific CSS attachments.  This feels out of scope for us.

**Comment by patrickbrown-io** [comment]
Cannot replicate on Windows -- testing on Windows 10 with default Dark Mode in Edge works fine. Tested both settings with using explicitly set Dark Mode as well as choosing the setting that adopts the System Settings and this is the same result:

Also tested the stock 'Page Colors' options in addition to Dark Mode to see if stacking options resulted in this and still could not replicate. Will test Dark Mode on Mac Edge as well.

![dark-mode-win10](https://github.com/user-attachments/assets/0de7d4aa-a563-42a8-b89a-b7936ed78197)

**Comment by patrickbrown-io** [comment]
Tested on MacOS as well-- cannot replicate

<img width="1728" alt="Screenshot 2025-01-07 at 10 32 31 AM" src="https://github.com/user-attachments/assets/ee57bec0-5866-464d-b75e-182daf59d307" />

**Comment by patrickbrown-io** [comment]
Closing as not planned, cannot replicate
